vault/core.go

```diff
@@ -2362,6 +2365,11 @@ func (s standardUnsealStrategy) unseal(ctx context.Context, logger log.Logger, c
 		if err := c.setupAuditedHeadersConfig(ctx); err != nil {
 			return err
 		}
+
+		if err := c.setupCensusAgent(); err != nil {
+			c.logger.Error("skipping reporting for nil agent", "error", err)
```

Comment:
Won't this code run in ent as well?  How do we know this is a "nil agent"?

Reply:
Good question. If `setupCensusAgent` errors  out, it will return a nil agent, so the nil assertion still holds.